packages/lib-classifier/src/store/Classification.js

```diff
@@ -19,6 +19,7 @@ const ClassificationMetadata = types.model('ClassificationMetadata', {
     already_seen: types.optional(types.boolean, false),
     finished_workflow: types.optional(types.boolean, false),
     retired: types.optional(types.boolean, false),
+    selected_at: types.maybe(types.Date),
```

Comment:
I think this will need to change to `types.maybe(types.string))` too?

Reply:
Should be fixed now.

Reply:
Should be fixed now.